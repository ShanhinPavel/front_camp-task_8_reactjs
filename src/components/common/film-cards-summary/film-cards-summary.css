.text {
  margin: 0;
  color: #ffffff;
  font-family: 'Open sans Bold';
  margin: auto 0;
}
