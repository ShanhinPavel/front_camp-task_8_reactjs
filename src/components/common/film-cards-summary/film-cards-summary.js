import React from 'react';
import PropTypes from 'prop-types';

import styles from './film-cards-summary.module.css';

export const FilmCardsSummaryTypes = {
  SAME_GENRES: 'SAME_GENRES',
  AMOUNT_FOUND_FILMS: 'AMOUNT_FOUND_FILMS',
};

export const FilmsCardsSummary = ({ filmsSummary }) => (
  <p className={styles.text}>{filmsSummary}</p>
);

FilmsCardsSummary.propTypes = {
  filmsSummary: PropTypes.string,
};
