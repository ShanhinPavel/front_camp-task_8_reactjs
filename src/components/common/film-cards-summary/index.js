export { generateStringWithGenreFoundMovies, generateStingWithAmountFoundMovies } from './utils';

export { FilmsCardsSummary } from './film-cards-summary';
