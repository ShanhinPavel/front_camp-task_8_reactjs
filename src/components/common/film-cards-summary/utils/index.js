export {
  generateMoviesAmount,
  generateMoviesGenre,
} from './generate-movies-summary';
