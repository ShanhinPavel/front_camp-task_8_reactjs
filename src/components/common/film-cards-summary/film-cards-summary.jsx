import React from 'react';
import PropTypes from 'prop-types';

import './film-cards-summary.css';

export const FilmCardsSummaryTypes = {
  SAME_GENRES: 'SAME_GENRES',
  AMOUNT_FOUND_FILMS: 'AMOUNT_FOUND_FILMS',
};

export const FilmsCardsSummary = ({ filmsSummary }) => (
  <p className={'films_summary'}>{filmsSummary}</p>
);

FilmsCardsSummary.propTypes = {
  filmsSummary: PropTypes.string,
};
