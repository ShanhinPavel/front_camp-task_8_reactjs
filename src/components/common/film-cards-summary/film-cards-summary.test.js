import React from 'react';
import { create } from 'react-test-renderer';

import { FilmsCardsSummary } from './film-cards-summary';

describe('FilmCardsSummary component', () => {
  let component;
  beforeAll(() => {
    component = create(<FilmsCardsSummary filmsSummary={'Summary'} />).toJSON();
  });

  it('Should render', () => {
    expect(component).toMatchSnapshot();
  });
});
