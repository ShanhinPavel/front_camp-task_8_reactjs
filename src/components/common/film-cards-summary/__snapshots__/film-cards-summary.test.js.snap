// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCardsSummary component Should render 1`] = `
<p
  className="films_summary"
>
  Summary
</p>
`;
